meta-ostro-xt/recipes-core/libxml/files/CVE-2016-5131-use-after-free-in-xpath.patch

```diff
@@ -0,0 +1,43 @@
+From e3edfa4fe989a90e9c768de88614de71538c8bb5 Mon Sep 17 00:00:00 2001
+From: Ismo Puustinen <ismo.puustinen@intel.com>
+Date: Wed, 7 Sep 2016 17:51:41 +0300
+Subject: [PATCH] CVE-2016-5131: use after free in xpath.
+
+CVE: CVE-2016-5131
+Upstream-status: Backport (from Chromium's fork of libxml)
```

Comment:
@ipuustin I wonder if this https://git.gnome.org/browse/libxml2/commit/xpath.c?id=d8083bf77955b7879c1290f0c0a24ab8cc70f7fb is the upstream fix for this?